server/views/components/media-item/index.njk

```diff
@@ -0,0 +1,21 @@
+{% if media.mediaType == 'image' %}
+    {% component 'figure', {
+    type: 'image',
+    icon: 'Picture',
```

Comment:
I _think_ @gestchild updated all of the icons to be lowercase now (`icon: 'picture'`).